libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/Microsoft.Bot.Builder.Adapters.Webex.csproj

```diff
@@ -19,6 +19,10 @@
     <DocumentationFile>bin\$(Configuration)\netstandard2.0\Microsoft.Bot.Builder.Adapters.Webex.xml</DocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(SignAssembly)'=='true' ">
```

Comment:
Wouldn't it be good to define this constant in Directory.Build.props (at the root of the solution) so we can take advantage of this setting across the board?

Reply:
That could be done, in which case it should go in as:
  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU' ">
    <SignAssembly>true</SignAssembly>
    <DelaySign>true</DelaySign>
  </PropertyGroup>

  <PropertyGroup Condition="'$(SignAssembly)'=='true' ">
    <DefineConstants>SIGNASSEMBLY</DefineConstants>
  </PropertyGroup>

If that is OK with you, I am willing.

Reply:
Let's do that so it is easier to add internal tests in the future.
Thanks

Reply:
OK, I added it. You approve?